src/syscheckd/src/db/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.12.4)
+
+project(fimdb)
+
+enable_testing()
+
+include_directories(${SRC_FOLDER}/shared_modules/utils)
+include_directories(${SRC_FOLDER}/shared_modules/dbsync/include/)
+include_directories(${SRC_FOLDER}/shared_modules/rsync/include/)
+include_directories(${SRC_FOLDER}/syscheckd)
+include_directories(${SRC_FOLDER}/syscheckd/src/db/src)
+
+file(GLOB DB_SRC
+     ${CMAKE_SOURCE_DIR}/src/db/src/db.cpp
+     ${CMAKE_SOURCE_DIR}/src/db/src/file.cpp
+     ${CMAKE_SOURCE_DIR}/src/db/src/fimDB.cpp
+     ${CMAKE_SOURCE_DIR}/src/db/src/dbFileItem.cpp
+     ${CMAKE_SOURCE_DIR}/src/db/src/dbRegistryKey.cpp
+     ${CMAKE_SOURCE_DIR}/src/db/src/dbRegistryValue.cpp)
+
```

Comment:
We are adding the registry database handling code to all the platforms, though the registry monitoring is only for Windows. Is there any reason for this? If not, I think that the best thing will be to avoid this behavior

Reply:
I will see to apply a factory pattern so that it is not necessary to compile this!